precompiles/stateful/IBCTransfer.sol

```diff
@@ -2,10 +2,10 @@
 pragma solidity >=0.8.17;
 
 /// @dev The ICS20I contract's address.
-address constant ICS20_PRECOMPILE_ADDRESS = 0x0000000000000000000000000000000000000802;
+address constant IBC_TRANSFER_PRECOMPILE_ADDRESS = 0x0000000000000000000000000000000000000802;
```

Comment:
Hi @Vvaradinov, forgot to change the variable name bellow, still `ICS20_PRECOMPILE_ADDRESS`